lib/phase-list.js

```diff
@@ -75,12 +77,12 @@ PhaseList.prototype._resolveNameAndAddToMap = function(phaseOrName) {
   }
 
   if (phase.id in this._phaseMap) {
-    throw new Error('Phase "' + phase.id + '" already exists.');
+    throw new Error(g.f('Phase "%s" already exists.', phase.id));
   }
 
 
   if(!phase.__isPhase__) {
-    throw new Error('Cannot add a non phase object to a PhaseList');
+    throw new Error(g.t('Cannot add a non phase object to a {{PhaseList}}'));
```

Comment:
I think you have to run slt-globalize -e again for intl/en/messages.json to get updated again with the brackets change.